Automattic/woocommerce-payments
# fix: address normalization on checkout for tokenized cart PRBs

Fixes part of https://github.com/Automattic/woocommerce-payments/issues/8841

#### Changes proposed in this Pull Request









UK and CA addresses couldn't place an order with tokenized cart PRBs.
Their addresses were failing zip code validation.

Turns out, that the address normalization for zip code shouldn't be needed on checkout - it's only needed when customer data is updated.

So I made a change to ensure the postcode is "normalized" only on route update, and nothing else.

#### Testing instructions





- Enable the "Enable Cart-Token implementation for PRBs" flag in your dev tools (you'll need to update them)
- Ensure you have Google Pay/Apple Pay enabled in the merchant's settings
- Go to a product page on your site
- Click on the PRB button
- Select a CA or UK address from the shipping address list
- Clicking "Pay now", you should be able to place an order

-------------------


**Post merge**
